test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala

```diff
@@ -42,6 +42,10 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
     }
   }
 
+  @JSExportTopLevel("org.scalajs.testinterface.HTMLRunner")
+  protected def getInstance(): this.type = this
+
+  @JSExport
```

Comment:
Replace this by a single top level main? We can do this safely, since we always pull `testinterface` to the newest version. So it always matches the sbt plugin version.